google/cloud/security/enforcer/gce_firewall_enforcer.py

```diff
@@ -17,6 +17,8 @@
 Simplifies the interface with the compute API for managing firewall policies.
 """
 
+# pylint: disable=relative-import
```

Comment:
You can disable this globally in the pylintrc file.
https://github.com/GoogleCloudPlatform/forseti-security/blob/master/pylintrc#L59